packages/server/src/notification/notification.controller.ts

```diff
@@ -39,11 +51,23 @@ export class NotificationController {
 
   // TODO: GET RID OF THIS
   @Post('/notify_user/:user_id')
+  @UseGuards(JwtAuthGuard)
   async notifyUser(@Param('user_id') user_id: number): Promise<string> {
     await this.notifService.notifyUser(
       user_id,
       "bruh if you're still tryna test notifications, ligma balls, FUCK FUCK FUCK FUCK FUCK, sugondese, hot cheetos",
     );
     return 'notified';
   }
+
+  @Post('/phone/verify')
+  async verifyPhoneUser(
```

Comment:
Can we use `@Body` instead of getting the request?

Also note that we can set the content type header using a decorator https://docs.nestjs.com/controllers#headers

Then I don't think we need the req and res at all

Reply:
we need res to send a message back right? i'm not too familiar with express/nest conventions and crossing still

Reply:
Nest lets us just return the data we wanna send.
For example:
https://github.com/sandboxnu/office-hours/blob/f3147bb0fb51674cb5565277255ab329d40fbe8b/packages/server/src/course/course.controller.ts#L21-L22